schemas/common/dataImportEventTypes.json

```diff
@@ -10,7 +10,9 @@
     "DI_INVENTORY_INSTANCE_CREATED",
     "DI_INVENTORY_INSTANCE_MATCHED",
     "DI_INVENTORY_INSTANCE_NOT_MATCHED",
+    "DI_INVENTORY_INSTANCE_UPDATED_READY_FOR_POST_PROC",
```

Comment:
POST_PROCESSING? Let's not cut the words in this way to avoid ambiguity

Reply:
done